core/ui/src/main/kotlin/nl/q42/template/ui/compose/ViewStateStringResolver.kt

```diff
@@ -0,0 +1,31 @@
+@file:OptIn(ExperimentalComposeUiApi::class)
+
+package nl.q42.template.ui.compose
+
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.ExperimentalComposeUiApi
+import androidx.compose.ui.res.pluralStringResource
+import androidx.compose.ui.res.stringResource
+import nl.q42.template.ui.presentation.ViewStateString
+
+/**
+ * Resolve a ViewStateString to a string from a Compose context.
+ */
+@Composable
+fun ViewStateString.resolve(): String {
```

Comment:
I would have personally named this function toString() to keep it familiar but this name is good enough.

Reply:
toString feels improper. It usually prints a description.

any other options? get() ?

Reply:
Yes, I like "get", short and sweet

Reply:
+1 for get(), clear in what to expect